NimbleMedium/Sources/Presentation/Modules/Home/HomeViewModel.swift

```diff
@@ -37,11 +37,24 @@ protocol HomeViewModelProtocol: ObservableViewModel {
 final class HomeViewModel: ObservableObject, HomeViewModelProtocol {
 
     private let disposeBag = DisposeBag()
+    private let unauthorizedNotificationTrigger = NotificationCenter.default.rx.notification(.unauthorized)
 
     var input: HomeViewModelInput { self }
     var output: HomeViewModelOutput { self }
 
+    @Injected var logoutUseCase: LogoutUseCaseProtocol
+
+    @PublishRelayProperty var didReceiveUnauthorized: Signal<Void>
+
     @Published var isSideMenuOpen: Bool = false
+
+    init() {
+        unauthorizedNotificationTrigger
```

Comment:
Do we need this property?
```suggestion
        NotificationCenter.default.rx.notification(.unauthorized)
```

Reply:
actually we can use directly like your suggestion